src/main/java/games/strategy/engine/random/MersenneTwister.java

```diff
@@ -68,10 +68,10 @@
   private static final int TEMPERING_MASK_B = 0x9d2c5680;
   private static final int TEMPERING_MASK_C = 0xefc60000;
   // the array for the state vector
-  private int m_mt[];
+  private int[] m_mt;
```

Comment:
heh, this is almost nostalgic, I had forgotten that the array brackets notation was optional in location.